### PR TITLE
Bundle size

### PR DESCRIPTION
**Pull request by mohe2015**
Fixes https://github.com/reform-org/reform/issues/323
```
Before:
✓ 485 modules transformed.
dist/index.html                     0.39 kB
dist/assets/index-1e9177cc.css    106.89 kB │ gzip:  17.27 kB
dist/assets/index-07a05a45.js   5,275.13 kB │ gzip: 940.13 kB

After
✓ 1535 modules transformed.
dist/index.html                                            0.39 kB
dist/assets/index-90651e86.css                           107.96 kB │ gzip:  17.39 kB
dist/assets/webapp.pages.HomePage__anon_1-fcd058dc.js    435.74 kB │ gzip: 180.26 kB │ map:  1,606.71 kB
dist/assets/index-6b3224f9.js                          3,095.32 kB │ gzip: 520.11 kB │ map: 11,695.40 kB
```

### CODE REVIEW

**Comment by mohe2015** [comment]
✓ 1519 modules transformed.
dist/index.html                     0.39 kB
dist/assets/index-c6c18839.css    105.76 kB │ gzip:  17.16 kB
dist/assets/index-3bef363e.js   3,903.11 kB │ gzip: 776.87 kB

**Comment by mohe2015** [comment]
✓ 1532 modules transformed.
dist/index.html                     0.39 kB
dist/assets/index-d7f43c34.css    107.64 kB │ gzip:  17.35 kB
dist/assets/index-b0a0411d.js   3,527.81 kB │ gzip: 700.12 kB

**Comment by mohe2015** [comment]
✓ 1532 modules transformed.
dist/index.html                     0.39 kB
dist/assets/index-90651e86.css    107.96 kB │ gzip:  17.39 kB
dist/assets/index-29e949a8.js   3,541.74 kB │ gzip: 701.77 kB │ map: 13,327.54 kB

**Comment by mohe2015** [comment]
✓ 1535 modules transformed.
dist/index.html                                            0.39 kB
dist/assets/index-90651e86.css                           107.96 kB │ gzip:  17.39 kB
dist/assets/webapp.pages.HomePage__anon_1-fcd058dc.js    435.74 kB │ gzip: 180.26 kB │ map:  1,606.71 kB
dist/assets/index-6b3224f9.js                          3,095.32 kB │ gzip: 520.11 kB │ map: 11,695.40 kB

File: utils.js [hunk 1]
```diff
@@ -120,4 +121,28 @@ export const createPopper = async (trigger, element, placement, sameWidthAsRef)
 	}, 100));
 };
 
-export const isSelenium = import.meta.env.VITE_SELENIUM == "true";
\ No newline at end of file
+export const isSelenium = import.meta.env.VITE_SELENIUM == "true";
+
+export const toGermanDate = (/** @type {number} */ input) => {
+	console.log("toYYYYMMDD", input)
```
Comment (lukasschreiber) [inline utils.js:127]: console log should be removed

**Pull request merged**